deployer/__main__.py

```diff
@@ -278,6 +307,13 @@ def main():
         parents=[base_parser],
         help="Deploy grafana dashboards to a cluster for monitoring JupyterHubs. deploy-support must be run first!",
     )
+
+    # setup-cluster-credentials subcommand
+    setup_cluster_credentials_parser = subparsers.add_parser(
+        "setup-cluster-credentials",
```

Comment:
Maybe consider calling it `use-cluster-credentials` or some such? `setup` makes it sound like it'll set up for general use by all shells, rather than just the shell we spawn.

Reply:
Done in [`b2c17aa` (#977)](https://github.com/2i2c-org/infrastructure/pull/977/commits/b2c17aa909a4434dccee637df5e59f06205269c5)